front/src/containers/modal-change-password/index.js

```diff
@@ -78,7 +78,7 @@ class ChangePassword extends Component {
                                     { validator: validatePassword }]} >
                                 <Input.Password placeholder={user === 'admin' ? 'Confirm password' : 'Confirm new password'} />
                             </Form.Item>
-                            <Form.Item name={user} style={{ display: 'none' }}>
+                            <Form.Item name="user" style={{ display: 'none' }}>
```

Comment:
Why do we need this input if it has `display: none`

Reply:
I use it for sending userType with form data